gg-pac-team3-back/src/controllers/userCourseController.mjs

```diff
@@ -0,0 +1,49 @@
+import admin from '../config/firebase.mjs';
+import UserCourseModel from '../models/userCourseModel.mjs';
+
+const db = admin.firestore();
+
+export const enrollInCourse = async (req, res) => {
+    const userId = req.user.uid; // Assuming you attach user ID to req.user in authentication middleware
+  const { courseId } = req.body;
+
+  try {
+    const userCourse = new UserCourseModel({
+      userId,
+      courseId,
+      enrolledAt: new Date()
+    });
+
+    const userCourseRef = db.collection('User_Courses').doc();
```

Comment:
Can we make this a constant like the rest?

Reply:
I did! Thank you